src/app/components/MainContent/index.jsx

```diff
@@ -41,7 +41,13 @@ const MainContent = ({ data }) => {
 MainContent.propTypes = {
   data: shape({
```

Comment:
We should remove this `data` prop and just pass `model`. Will then need to make the relevant change [here](https://github.com/bbc/simorgh/pull/251/files#diff-89fcabca58d80c9d5cba58347724d5ceR32), and then you can just destructure `blocks` out of `model` and pass blocks to this [render function](https://github.com/bbc/simorgh/pull/251/files#diff-89fcabca58d80c9d5cba58347724d5ceR33) and it won't need to [destructure blocks](https://github.com/bbc/simorgh/pull/251/files#diff-89fcabca58d80c9d5cba58347724d5ceR21)